user_guide/integrations.rst

```diff
@@ -0,0 +1,42 @@
+Integrations
+============
+
+Behat can be integrated with a large number of other projects and frameworks to enhance its
+capabilities. We'll mention here the main integrations that may prove useful in your projects:
+
+Mink
+----
+
+Behat can be used to describe the business logic of many different projects, but one of the
+main uses is for web applications where it can be used to provide functional testing. `Mink`_
+is a library which lets you control or emulate a web browser and which lets you simulate the
+interactions of users with a web page. It provides a number of drivers for tools like
+Selenium or Goutte to implement these capabilities.
```

Comment:
Possibly worth mentioning the Chrome DevTools driver (which AFAIK is not official, hence changing "provides" to "supports") partly because it's useful, but also because some new people might have heard of that technology but not yet come across Selenium / BrowserKit?

I'd also suggest listing BrowserKit over Goutte, which is now deprecated & archived.

```suggestion
interactions of users with a web page. It supports a number of drivers for tools like
Selenium, BrowserKit, and Chrome DevTools Protocol to implement these capabilities.
```

Reply:
Thanks, updated!